src/main/resources/io/jenkins/plugins/appcenter/Messages.properties

```diff
@@ -1,9 +1,11 @@
 AppCenterRecorder.DescriptorImpl.errors.missingApiToken=Please specify an AppCenter API Token
-AppCenterRecorder.DescriptorImpl.errors.invalidApiToken=API Token canot contain whitespace
+AppCenterRecorder.DescriptorImpl.errors.invalidApiToken=API Token can't contain whitespace
```

Comment:
You're correct, this was a typo on my end but could you change it to: `cannot` please. Sorry, I know it is minor, but that was what I was intending.

Reply:
Sure 👍